app/src/main/java/net/gsantner/markor/format/asciidoc/AsciidocActionButtons.java

```diff
@@ -0,0 +1,541 @@
+
+                //similar to other formats:
+                new ActionItem(R.string.abid_common_delete_lines, R.drawable.ic_delete_black_24dp,
+                        R.string.delete_lines),
+                new ActionItem(
+                        R.string.abid_common_open_link_browser,
+                        R.drawable.ic_open_in_browser_black_24dp,
+                        R.string.open_link),
+                // a different icon was used than in other formats, so that you can distinguish
+                // it from opening in the browser
+                new ActionItem(R.string.abid_common_view_file_in_other_app,
+                        R.drawable.in_open_new_ic_icon_24dp, R.string.open_with,
+                        ActionItem.DisplayMode.ANY),
+                new ActionItem(
+                        R.string.abid_common_attach_something, R.drawable.ic_attach_file_black_24dp,
+                        R.string.attach),
+                new ActionItem(R.string.abid_common_time,
+                        R.drawable.ic_access_time_black_24dp, R.string.date_and_time),
+                new ActionItem(
+                        R.string.abid_common_new_line_below,
+                        R.drawable.ic_baseline_keyboard_return_24,
+                        R.string.start_new_line_below),
+                new ActionItem(
+                        R.string.abid_common_move_text_one_line_up,
+                        R.drawable.ic_baseline_arrow_upward_24,
+                        R.string.move_text_one_line_up),
+                new ActionItem(
+                        R.string.abid_common_move_text_one_line_down,
+                        R.drawable.ic_baseline_arrow_downward_24, R.string.move_text_one_line_down),
+                new ActionItem(R.string.abid_common_web_jump_to_very_top_or_bottom,
+                        R.drawable.ic_vertical_align_center_black_24dp, R.string.jump_to_bottom,
+                        // doesn't work in editor:
+                        // ActionItem.DisplayMode.ANY),
+                        ActionItem.DisplayMode.VIEW),
+                new ActionItem(
+                        R.string.abid_common_rotate_screen, R.drawable.ic_rotate_left_black_24dp,
+                        R.string.rotate, ActionItem.DisplayMode.ANY),
+
+        };
+
+        return Arrays.asList(TMA_ACTIONS);
+    }
+
+    /*
+     * DONE: indent deindent
+     *
+     * Maybe two different indent deindent
+     *
+     * a "normal" one for blank characters only
+     ** If the pattern is not filled:
+     *** indent: blank as prefix
+     *** deindent: remove a blank
+     * another one that will change the level of headers and lists +
+     * Pseudologics for the level changer:
+     ** If the pattern is satisfied for headers or lists: leading #, *, . with the
+     * following
+     * blank
+     *** indent: the first character is added as a prefix
+     *** deindent: the first character is removed, but only if something remains after
```

Comment:
Note that my automatic list ordering system only works with actual indentations based on spaces / tabs. It will not work here.